examples/userapi/01_dsl.ipynb

```diff
@@ -675,19 +681,28 @@
       "} ;\n",
       "\n",
       "\n",
-      "int Kernel(const float dt, const float h_x, const float h_y, struct dataobj *restrict u_vec, const int time_M, const int time_m, struct profiler * timers, const int x_M, const int x_m, const int y_M, const int y_m)\n",
+      "int Kernel(const float dt, const float h_x, const float h_y, struct dataobj *restrict u_vec, const int time_M, const int time_m, struct profiler * timers, const int x_M, const int x_m, const int y_M, const int y_m, const int nthreads)\n",
       "{\n",
       "  float (*restrict u)[u_vec->size[1]][u_vec->size[2]] __attribute__ ((aligned (64))) = (float (*)[u_vec->size[1]][u_vec->size[2]]) u_vec->data;\n",
+      "  /* Flush denormal numbers to zero in hardware */\n",
```

Comment:
have you reinstated optimizations here?
 I **think** they were off on purpose (since it's an early tutorial?)

Reply:
I haven't touched anything I only reran it because the generated code is slightly different.